lib/controllers/ephemeral/account-linking/associated_accounts_controller.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:collection';
+import 'package:pispapp/models/consent.dart';
+
+import 'package:get/get.dart';
+
+class AssociatedAccountsController extends GetxController {
+  AssociatedAccountsController(this.associatedAccounts);
+  final List<Account> associatedAccounts;
```

Comment:
could this be private?

Reply:
It could have been but then I changed the list items to be built using that. I feel like this controller is closer to the screen than the flow controller so now it should be exposed I think.